pkg/filter/util/cloudcost.go

```diff
@@ -53,6 +53,61 @@ func CloudCostFilterFromParams(pmr mapper.PrimitiveMapReader) filter.Filter[*kub
 	return filter
 }
 
+type CloudCostFilters struct {
+	FilterInvoiceEntityIDs []string `json:"filterInvoiceEntityIDs"`
+	FilterAccountIDs       []string `json:"filterAccountIDs"`
+	FilterProviders        []string `json:"filterProviders"`
+	FilterProviderIDs      []string `json:"filterProviderIDs"`
+	FilterServices         []string `json:"filterServices"`
+	FilterCategories       []string `json:"filterCategories"`
+	FilterLabels           []string `json:"filterLabels"`
+}
+
+func (g *CloudCostFilters) Equals(that CloudCostFilters) bool {
+	// FIXME
+	return false
+}
+
+func ParseCloudCostFilter(filters CloudCostFilters) filter.Filter[*kubecost.CloudCost] {
```

Comment:
this function is copy / paste. I plan to refactor the existing implementation into a transform and use this for the logic, to remove duplication.